system-int-test/test/cmr/system_int_test/ingest/subscription_time_frame_job_test.clj

```diff
@@ -0,0 +1,92 @@
+        (is (= 401 (:status response)))
+        (is (= ["You do not have permission to perform that action."] errors))))
+
+    (testing "returns 401 unauthorized when regular user token is provided"
+      (let [response (client/post (url/email-subscription-processing)
+                                  {:accept "application/json"
+                                   :content-type mt/form-url-encoded
+                                   :body (codec/form-encode {:revision-date "2000-01-01T10:00:00Z,2010-03-10T12:00:00Z"})
+                                   :query-params {:token non-admin-token}
+                                   :connection-manager (system/conn-mgr)
+                                   :throw-exceptions false})
+            errors (:errors (json/decode (:body response) true))]
+        (is (= 401 (:status response)))
+        (is (= ["You do not have permission to perform that action."] errors))))
+
+    (testing "returns 200 with valid time range format"
+      (let [response (client/post (url/email-subscription-processing)
+                                  {:accept "application/json"
+                                   :content-type mt/form-url-encoded
+                                   :body (codec/form-encode {:revision-date "2000-01-01T10:00:00Z,2010-03-10T12:00:00Z"})
+                                   :query-params {:token admin-update-token}
+                                   :connection-manager (system/conn-mgr)
+                                   :throw-exceptions false})]
+        (is (= 200
+               (:status response)))))
+
+    (testing "returns 422 with invalid time range format"
+      (let [response (client/post (url/email-subscription-processing)
+                                  {:accept "application/json"
+                                   :content-type mt/form-url-encoded
+                                   :body (codec/form-encode {:revision-date "2000-01-01T10:00:00Z,2010-03-10T12:00:"})
+                                   :query-params {:token admin-update-token}
+                                   :connection-manager (system/conn-mgr)
+                                   :throw-exceptions false})]
+        (is (= 422
+               (:status response)))))
+
+    (testing "returns 400 when only a start-date is provided"
+      (let [response (client/post (url/email-subscription-processing)
+                                  {:accept "application/json"
+                                   :content-type mt/form-url-encoded
+                                   :body (codec/form-encode {:revision-date "2000-01-01T10:00:00Z,"})
+                                   :query-params {:token admin-update-token}
+                                   :connection-manager (system/conn-mgr)
+                                   :throw-exceptions false})]
+        (is (= 400
+               (:status response)))))
+
+    (testing "returns 400 when only an end-date is provided"
```

Comment:
Consider using `are3` to simplify the tests for the various revision-date-range error cases.